packages/editor/src/pages/Editor/components/Backstage/GalleryList/index.tsx

```diff
@@ -34,6 +35,7 @@ class GalleryList extends Component<IProps, IState> {
               <Icon iconName={isExpanded ? 'ChevronUp' : 'ChevronDown'} />
             </ArrowWrapper>
           </TitleBar>
+          <Only when={messageBar !== undefined}>{messageBar}</Only>
```

Comment:
Out of curiosity, could it instead be `when{(messageBar)}`?

Reply:
nope. because ReactElement is not assignable to Boolean, or something like that